### ISSUE TITLE
Condition activates for a wrong aura for a split seconds

### ISSUE THREAD

**Issue by Loradim** [WeakAuras/WeakAuras2#2155]
## Describe the bug

`!TR1EZTnUr8VlPtNXEAIp9Yo5M2CZi5hXkXNKROC9D9pKiejKesOiviiTJ8mLF27Ula4lr9kNV0BAUX2sKGlwSyFH93spQ(OHJSKJSACs9xFsRrwlgz5I)cJoH58j3WGLNh4fecd0Ro9Bd6ZM0NTg1R2jNokcioi0Lh2rpLrwD8ep9el0nXEyqGxKag6l9NovYJGPyiF0eZvx6oJpYY6ZXSqEI9vXEEj23pxebdY8DMheEBGWhwMZVS3WlhyMLL4jokq6BZlO1ibTgjO1ibTokMrbopWdLIa)r9EtTM1arHfhb8V)YiyqjmBGkMJ5glzelmsnQWxOVAkCPCoDnYZqXSzatvR8W0bWlD5tINoD4QLWg56lV52RU7g450TS4qgi6YLCpVUU6vwgpH)a3pYcML4lJSgFEBRHJTg2EaWSyuaSw6XwbmpL0Bd5ePw3E5naZPXGfJZ8IMpYYNTGlnsASFMOfPgAvoJcOH5H(mV)Lrd14hhzfYNbxR2bZcdIbdPK7nvBp60F4W()8nxEfiEcWXPnSPsSFNxWJj2DINzSmD9P1OoO6cbbc0qSigDVY6EfoQAnSoFWLx2dwehMhNijgz8hNEuRp86H9ExN)THPgjVfPlgqYPXa6lwWqByEtOwV7h4dFYzsum4(ZaTe4Ky5csooJXkQKCNaFWQqeomBIaZxWe(pJSl1z6zIHalXhjY5p7fWOiAWxZb)ErmerACkC8ys5AJkXqRsdQ48uXS0PM6YxrquEpRkJHDGedUbp6B9Oae)ozdqjdM8CggQ0zoXYOGf7iiRu8ZgIYkg1Ay8REvID8sxweKddMv4QeBYFpXgEY7Ng7tBOJoo59j2QFWzmJhLyhjG40i2ILVmXwgKy)imP5ShGpJMdFijUKseW3GWeBgMM0zo35tsygIO5c)0vulhPRKxaeqPyHLIfVfcu5rdHbQuIe(WAOIIsSX)M5fmbzHXmTgVjJ07o)cyly8wv3X9zt84UQBwe4oaKm4gJiyHkZZ1C9OZQ3SwRJvcIB8IfGsK0269moVYl85xpEahdlf(Zm8Db7lhv7LzcvI9FRGyblqoTXXvXtGkHVLkqJ4z9tonX(hsSpQoXT7aprs2VMjJ4h9cvM5xCmrt9A1kRvzGPmScfRE7HgWmTRmBYghhszz0tDbUCentmfFULQkJsolsM5iZwuTHyBmN4knn9vOPP260zwvI0SB2a1Ow6((d(q7b9VRhyHScwWjFEDuq7B7MyhYJIdb1JTkaBUygO1MemlggYpEXeifaOfzXsWzaUWtarVyubNZFIlrnHaoBbRmb45cGb8q(jW6g8yBhyneszm81FbSFVzDjumfS(z7JFkX(SATEdAQrU5V(eQuru7VVzcZ0Svrf33D3QT7dUh8MczlbfbdoYgMJTphC)ey(KhyEXCPXDRrY)PzJFa2S10jEWDiM8bsUctJLlC3gZUq2HPSq0ReSd8xQ342WQeOFmU0lrTF1Qp9o8NYN(bdNqrPUj2z3A01CbFLE3KYI9u9HcvbtATDT0fNbdvuz3(pmjkqN8C(kV9TLY)qZZSlEBwMQDUZn)OIeGv7LKmJUbeJfOD3bT2WdMedS3mAiN5UsjzpkWCotaZfBAeVCI11uCEs(UfiDkNqUZCw4mE3cjplKqUq24nMkEhRJAv0PMjMuXk)d5wN9wJQ58lRKJOEgwFNypk1zQuq2td9BrvwLp421VvyO95c1PjKf3hTOfn97W5VWnzh5rNam(dc8yGflGD04ZtvffES18Gi4qx9MvF(rHZMEXhipmnFErEZA6vJF3jaxppysiJyype3aWRJw)eDmH4XLN5qOCO0jSrzhcQrpKCZ2OqNYi9Usza)ykLZuKDelkgQXMkXcaIqLuLhtvjizvH2Rcyxwkl0q1G)CGlGZ5v1RPkUTciyFIZx2gRDpAa6hJLgNdcMPCAxUeK2qucNucrKUEy(xIglNZal0VKR2CAy8Jrw)vzbQ2h80e1zct77g2xpkajiaRYX5EHlwjDB4E9JGkP5UQH7Dwld8GFTSq9rWwiMUkBlggage(qsw71V3LMc7HnRAdqtZzR9RitSNgGs8vHINsS)NXmxetAI9WIAQ1eQhbSN3dhWnY6(0ROh8GqkGYQqGmQ9FEmpD717YbJvOLh09DxpmNmO6Ir9gg9WAghKiLtv)7gEtxyFtErZ5qPprkn4ZhcV849fqgVra8Sudv(w4qpmhQn0SCHqsLwsO3q7DNkB5swNcWggbrFhqFb2dRBoqQqafU4uiXtbagnuYPgk4(5AD7BvVHR8xxhWmZB5CJC89CNeIqf8W9sP)7fC)CG8ZG(VRUUP3HJSal4mUppu4aNV8yFyej8DBVhzRKBeC)wAAN5eHyFDFciVCRyjFCX1Pt5Zg00FEA103ggmlKlHtNoIo56yfBOz1bJdzE049GqGrwfomvrPm9PDwRnIPpQxZw1AC2UASiWn2ezq4e6WOoQrGGr4uVoBPhIBr3M57SwBy2Go8aA9zb7RMjT1qKVJYa9BsxUU6ydokvA2RqN03Faw4Dj16AgP9q7A4ePEBws9cCp1DyxMBJQPkvj2xdq)afpbHKGeaNc7i3Qsa9qu58dzgVfm75TGblX(cqKZvSepQspU1JOq6fsAwtk1189OsRClhPTA9n0z8Rnk)E2NGIPVB5FiCkBGyORDWELNU1EVA(oFZqnvJNRM(3tffSvRqPIYRSW99QzUL3GgruLVZvi)ijRpqVlQvvw9E9dP29QlPyTJPZFg(HxsXZe7YkP45HHrfWP05pXP8)74uu1aOEHDJd0tOrofI(nbBiXz32snL52h6r40lro7np32zUaI3xqna((Ga3ukjSp9ol9q0zEbpEvi)ZXCFNv470UXP5er8PuAs89AQC2MOMJA3El4M4Pg4gD8bGMNUpVzzMxrx1nyZPvtZ)FP00hox48jF44eshIJyL(AsPLxapeGzPURJ5LSxaUyZgzWfXu)k0HDspYXHQn3aze7Iowo113PCTlcHuJ2utekfFDqmZJRSIwBKb1mhmnvYc1Pa5oXP3RG93dSqHQmW29Uqt06if6vGs8Gm8EVyDc9ILVIP82aX1sF7OZz(Z4zl0YWGL8WOvfs6vax13mPFhexOus8nQVfIRCRQE(KTVNBw9E(0TUCByPqNTS1sby3SRvu0jpbqy8jnpHITIm7HY9Cq3GGtAQRcCYgBEXfc5sQ02CDBSyFh0DqO2jV(00oOGrEuVhUWV3fnxEtN76oiRdfDQ(FfLn1hexLi0fdn6vVzJ6V(Smi1BPVmgZqAixZwphG3nHOBdw((wzQbE(UBUAPI4pqqUB))GMTwdFUwaxQeEQz2j2y7Spey61)XMTo901WHuaCrHc1ZHI4zfv(oq(ujiTQXNSvmhfmThKkSsVOn1jN9Srbvb)rBr(Tck)qH)u6LHGVyNS3sX)JGe9NaH(dkqO)eg0VpWGm7GVBX(yaH8Tc0tjSwFpH95WETh5QrDVqd14WQT(RaXsZVMQ33zT5vGcCRVOX9Ow90twlwPEzua5kwVI6XP63h86WJ)CRBUU7QGI1VVPI1lGgOqL7ToRE96Tkx5(UIxiv0dW1TAG3o6)o`

I made 3 Weakauras:
a) one Aura Backend, which features custom code to set certain global variables
b) one "Kill Command" aura, with a condition that glows the WA once a variable is set (Kill Command is ready)
c) one "Cobra Shot" aura which glows when another variable is set (GCD is ready)

Use Kill Command (both variables are set to false). Once GCD is ready again, variable for c) is set to true. Once Kill Command CD is finished, variable b) is set to true

Expected Behaviour:
If condition for c) is met, only "Cobra Shot" WA should glow

Behavior that happened:
Once variable for c) was set, for a splitsecond aura b) would be activated (wrong), then aura c) gets activated correctly. To me it seems, the positioning of the glow is incorrectly set and is only set correctly one frame later.

EDIT: as mentioned in my comment below: It appears not the positioning is wrong, but instead the wrong aura gets activated for a splitsecond.

**Do you have an error log of what happened?**
There are no LUA errors.

### To Reproduce
1. Import auras as provided above
2. Play a hunter and use Kill Command
3. Look at the auras and wait until GCD is ready
4. You can see that Kill Command will glow for a splitsecond, after which Cobra Shot will glow correctly

**Screenshots**
No screenshot, as it only happens for a frame or so. If needed, I can try to provide a video instead.

**Did you try having WeakAuras as the only enabled addon and everything else (especially something like ElvUI) disabled?**
Yes, only "Weak Auras 2", "Weak Auras Archive", "WeakAuras Model Paths", "WeakAuras Options" and "WeakAuras Templates" were loaded

**Which version of WeakAuras are you using?**
2.1.74

**Are you on World of Warcraft Classic or Retail?**
- [X] Retail

**Was it working in a previous version? If yes, which was the last good one?**
Can't tell :(

### Additional Info
-

```
  ERROR HERE
```

**Comment by Loradim** [comment]
I just did a bit more testing and changed aura glow for "Kill Command" to another type of glow: "Autocast Shine", while keeping the glow for "Cobra Shot" to be "Pixel Glow".

What did I expect:
Once the GCD was ready, I was expecting "Cobra Shot" to glow with "Pixel Glow".

What happend instead
For a split second "Kill command" did glow with "Autocast shine", then a frame later or so, the correct "Cobra Shot" with "Pixel Glow" would appear

I could not reproduce it with "Action Button Glow", but that might be, because of the animation of that kind of glow

**Comment by mrbuds** [comment]
Can you give a minimal example without your custom code ?

**Comment by Loradim** [comment]
Unfortunately, I wasn't able to reproduce the issue without any custom code. However, I managed to strip down the necessary custom code to about 10 lines and still have the same (wrong?) behaviour:

`!TNr2YTnYXVMuLuL1Q4LK29H8ajn1XUYGkeqr7MhaXqGHKJfigAmden1d4Bp9CHBEiBfLuExlzkIgn6UN(UB4221X1M5A35S2xEwpx7vU2bIFbOZq(pfetxpKgsJbawTL)2r(zx5N9CTAD25UCazACaoEG(rCThesE5fuCqQNdLgYjaOVoE(CgMdpIbD3zU2BlangVGqJC2Ug7AViMMS2GOn5fSGZ6llkrTKsulPe1skrTfYdN6)moMbKZ16NB1TfqDucFjnE8AoaKbpnGfY3CHnJJI5kOZjre2s13HVQGkOzmzXcGOko7Kbq81a8SK5ZvI(nJU7(RE4o4(YlrjXiq0zRXHH3gO5mlzg(zCe3gEkYxDTNoSVTZuBN(taILiyQ96q0wG4zOEFmwIQ99JUdiUegWmmkKV01ocTcZmsAsuUOXvGuAUrblajY(ljOyCQ3vjHHPEpUKWXgemwdWvaf5d6R7PebBgoYYz0eqyWHZ1Ggm2Xz8NUB0vGetaNM(W5m176q6MuVbjlmu82ijbBd61yqgbLgIJKxRyWvcOkfN9WjJgzbmXhfILOKii8NNFsVF7shRRh8Vfgeooocf(Vmw3o)Iufnr67ySRrKviHPTOL1gJyc2JJwakmWFrBFIOrWNbG0lEIPkGmSpncmwYhYjhpG4RqKO3qYL5T9grqGKIBrk4MhsrYiAWf0x83vjqePXxXpeXy1GYerCvaQO8CYIShnlsOyuCJ(n(qwGa6Mi7neqwhKdq6rc5biaPfW3xqM6O7NW40vhieQs0XoIHmK6dFi1lzDaIdbfaEXBt9KoRPEWD(15jrY8eNCA6VM6P(r8elW8upobI74OvR)Pupgn1Bd8qlrpdFYxcFWKujdjGU04upKiUZFj2)jg8ee(ssugh1YrgNcPq0GIe2ks8pGOmm3ba0OerIaEOC)t9e)FriDMGegvEnAlJqUE4hHJGXntDfocnlehOUyfnycizWfgrWwKsBOMQNCr7UT6DQsqcswTcuIYCE6ZS45QY4H3mDcweprIwyO7k0xpP1pLluPE)9sIfWGcAJckGPxFgqprwOH0zXi7LuUKGNuNla0wfEYympjwybqHmqZJJcG7b)unPD58Ys7hauAVkMaw7wzIu5iEcR5kenKQwwt6zSJc4NObqIWp0ULQwudP(EcJx3xeyZNi0qIqP)KN9laZanESWOmRsPbDgf8x5tzlrGx7VxiVLeS4dx7)gRewht)gsSZlo2)bNXAOqFhuH9W)rsGixuF4A9TGCr4afyRlYk5(hvfQpd(tK5BZZMgtbNkSJuwTgBnYOSGdR6aiFm)924wUypNkK4RIjVK69ptqbIc0qFBL1u1eQnq19hJrqpApM9n5nEMWiqwdzYC55Vy9GBTSgnzQQ1Hj3E9nofKbvxET7y0d1mocKuw7Xp4C3TW5wAWxIjlwYvAWVHsDjm80irwFP7sZ19k2zQSs1UBEPwxo1B78fku0Xs1skvQNQ3TAEhvIMQHus5Lmf29psyYuZY49W1lv8AtURuXUTLDrf8Lw3o2UDNa2FisLi4Nma5i8sku1VwVfVUwOleHO9L5cw6uwEFF6Mqy3rZy04zYSSa2lWr4yIpu5yZyWeZG)2pCdAlRu2990QVjx)(7Uh4Bzon4qn5BQXM6DFmDrmMbTnCIS67Pk6jRmnq4NIcLWTa511(3iIYVdPRwHeoqcmzz3DqTzsYULv3ET6CHO4KwaKoQzzHDTBRig4LdLahuPYyLQA7rTYZm)16YZOk1cqFFv)mpiD7pGgEhkZJYGpoAcgfSTAX(xPITrBCrdvzvzz1CjnBnJYrODnNbP6TBJQx9qPnW5gLqnwpa3do2lrG0b9IqGYs(S9QVfXn6YWhCExmVjBKqsnrFICC3dwKuVpchqtYBXJtysImtcXyfoIwUkWBP2Q3X4mU3WX)h7z(i6jyiIhw)E5u2PDl4FVAVYZRU)KISEFh(DObRN7QSLRXwYBuHCuPClRB(gsou2AVxFMkZzu26xolCUT8vApSdiSplhZ9zrYHOTnohs7xZui1l1xSZ8b)vN5)O3zUQWUAjGtL9vBDrb1HEfWge8pSLuJzHtHgcwUvZ8vo33FjbINwbruPEpsPbzys1YDNSkJlcPBUkg)LeCK)wX6S7CUcOUwIyFMkVSzk4QJz)H2ljIjtfGUtp0kmMU86IwKfHL9s3L52OwKSw(u)EfQ4SK4)ueKisUjCbe7S9KkLcqEG79ZQRgy2WEPrK62jFejr2c1erdQmeKoJLVovVCqkgjAHMznptLgb7BEqfjCmRtSPXQ(t(MBpMHXYhatL2uvHOqbvRpa25NrXevZY9T(OgP6dczvctrDbX1Hj609L7oxC02bYTYwr8su0cCoJwhtxJJ5BlL(NxO1)3pP)aixQtzXBazpi34rvD)z7)m3T5Z857LD7GvIOXCEP2dG5uRWyqreG0lN19mzggU5mO3fXzD3Z(feLfNTZTuaOUw2tUOVNSUEkUZa9ckAD2LNVNfN0WIsKlUGUEZZpE6lbFQZv5lUyq9fTOx(HifxGsKUvKsXQD3oTV8I8JNCOIxZwlmgTSvu0T3pa7OOH(7FN2kHC98PEIf0F4DsS3PPYB6V9V0T35N))vROObfCztzJQ8VHzFEvQ36Jp040FhCdwnnlO2i89UHIdplOXZR8bOGKiFxuvFHHnp)BfRun1tzB239gaQySUw)U5kFIkzXmIFXxdP5nw1478s822YFvw7y23QtvcCTWsHlBHB0TRP3D1B)uKn3pyX2RQ171RVFW3iYL3p4Bdb5Vjtm)xZl)FN5LnNG3IHK31qX5JR((oMSzA9VZHKVhCkcFphrU8aX1hy(hJrKBS31S(vl2f7XnrzPUL7Dr72T79MoxqwPjDd9tUm(0V07UBUDlTYSbfAHVHPbkosrH5ao63e7rn5CNx3CyFdt329BzsVdohxXPBA6TScF)zGI96iyI7)5d`

**Comment by InfusOnWoW** [comment]
It appears you believe that setting a variable in one aura should immediately affect another aura. That's not the case, it might not check again until the next frame.

Though frankly you have far too many triggers to call this a minimal example, please try harder to remove anything that is not relevant to reproduce the issue.

**Comment by Loradim** [comment]
> It appears you believe that setting a variable in one aura should immediately affect another aura. That's not the case, it might not check again until the next frame.

No, I'm not expecting that the correct WA Glow appears in the same frame. Showing up one frame later is totally fine.

The incorrect behaviour I'm trying to explain, that for a frame or so, glow appears **on a different Weakaura that it should**.

> Though frankly you have far too many triggers to call this a minimal example, please try harder to remove anything that is not relevant to reproduce the issue.

I'm sorry if I didn't think to further reduce the triggers, I'm happy to see how I can reduce it even more while still being reproducible. I'm more than willing to help track down this bug, but seeing this issue being closed without solution and without giving me the opportunity to provide better example irritates me a bit.

Would you like me to post this here or open a new issue?

**Comment by InfusOnWoW** [comment]
If you provide further information, then we'll investigate again. A ticket is closed if there's nothing for us to do at that time, it can be reopened at any time if more information points is added.

**Comment by Loradim** [comment]
So I checked further what is removable and still reproduces this error:

* I removed one aura and put necessary code into other aura
* I removed all unnecessary triggers and conditions for formatting the aura
* I reduced the custom lua code to the bare minimum needed to reproduce issue:
* Did remove some "fancy" formatting, cooldown swipe, etc

I can't further remove (because the wrong aura glow is no longer happening then):
* Text for both auras
* Border for both auras

So now both auras consist of only two triggers:
* 1st trigger to always show the aura
* 2nd trigger for aura "Kill Command" => Glow when Kill Command is not on CD
* 2nd trigger for aura "Cobra Shot" => Glow when no GCD

To see issue:
* use hunter, have no CD on GCD and no CD on kill command (both auras glow = Expected)
* use "Kill Command" (both auras stop glowing = Expected)
* wait until GCD becomes ready again
* now for a frame or so "Kill Command" aura will glow (wrong, because Kill Command is still on CD)
* after another frame or so "Kill Command" doesn't glow anymore, and now "Cobra Shot" glows (Expected)

Please let me know, if I can provide anything else to track down this issue.

Kind regards,
Loradim


`!TJvxZPnYs0FnBvovTXfcW4npCFa8IDSJd4fjhN9fHgKgGjwsdzgjSXpOF7B39ijKaSbFxFV1wBszxG08rpNP)4mNbxlxhxBTRDZJTo942U2rU2b4)qRty(3hOKlotgkvqddSO)BsF2I(ST7GghFIBcmyPkGR6Lpfx7EHINEIPcY8CKYWeb0K2NfYXPLpy3jU2RgoDQMNaMX1Mf7pxQUrkIb7Dw)bo9hvmuBXtCCLZFTkIAqiQbHOgeISq8Ki9xYvAHm2DWV1OvdW(pwDT8tGU00MpHPsGhGjjIf5pnfEupNEgnMsmBgynZs6u2a(yaFs60PoRwWDT)y)RV58BVg6NELLQyaM1l4HHxgOnwwNoHVKhNydZs8OR94Z6A7m22P7iWyPiaSxeYwbgVCO3O40qTVP)1GXP2GfJZctM7AhZI46cKMgVgAjMMmUS(bZaez)9uMIN5DEAyyM3DZfj8Ibu4CGCafFg4CmBPzkzkg74HtZJm9g64m8Zx3)CaVcixPlSlZ8Uiu(qMxV0zzEGf8LXtfZYDLkaGGhJLWOGVjmFo2QzjSpBu)(daSJwl(IOV9j)ZV6(K0cKDzmbmCQPjWuhUOi2zCNJi4QlcHjCvml8lfb)MFGGtGOYKcLmkdpvZh7hY0AmIMYPqLp2rukKYw4tnJaZueXZcXW8TMCZAJsJPOBm1eC3kIy4sxnvlp)iwgdFYzA09WJNbrtil2oaCO4mgBgLMJWxBgOZ6jcgpIjIFdnxzs)BKbnEVYIg)xKhHQAlPdo554d8b2Ka5dX2piGvQ36gOe8jpxLDL651v57PYgZoMXJ5kHV9C5ddJXLREd2A47UHpWwP3S49f4faCLKQ3dvW6YAciKX7vw8Fw(UoZ7gLCMIR1zEhzJenVBDaY12clMzHuhdad6A)jbw5FMmkIfhymTUS3EBrwv21GwTB0SdUjZraulknZNnrlvt(SmGma2cuS7FpzTDYGTbp1l4tXyeKO8FxCzy8ioly1wSQBTNQhv8t1jYOX1BSAiOM79qIh7pea8WraRusHdSwizh(46bb)5CWDBNUiGH85V8jmVjHNcxlvdfi0Flngk7wIN5fVYudUK7ymbbAt59ojNVNZx0fjEtgHUvQgENKMBrevLL61tA(gzU1KMVngeDOCnK1PWGzVnoHlNKI)yYy9CgKX91kIAOMXpCT)fDTrDicMOrV(y(U36mmVvmldJh(3jcqfhDPSoQliTHhyAEqN2fC2)5MG6BqnLy6Q104kjuyXDiSoy4G(LfHPtmBaAAhWjgguireFUs8uM3FKYcqTgGWZ6EQTa1dG6I7umqDZDLprDSuOftcjNVz)x9GOlhmO)OXgrqJU8Ip6ubdgzQwnl8dBfCWbzI2dV156lH9Dj)wX2pxBCU2NXYCluQ9TMt5zIL5JSY(iVfoPcCT28U(ZfadqeWbaYbLYGYrsYzg0bXxlIea03DUI)9uES)ku1FZtmnI1RaPcQ)ZKLnX0UzBEdKFeAA468cdRgM3RgpMfEy5OfojAHPz91nSIZCH)9Xa)lIXg5y0U8Mheqeq)WTcmV1R4UiuP3CUy2CCp3CxQeZjJr2xv((EmjzS3bRsmxGz(GqiDiQiRQix4Jh9ui7blH(DHMbX5aI6CBjY0e6THk4Cjz0jhvoEfZZwYuc0EgQE89W0cgC)5S4z81gW0hy8fWT(4QKvu454whtrOIK7D1PjCxKELVgn39yxhutk1vhUyodJXNcX4NKYi85w5NiozN3)dVEb4QqDazE1oeEV3z5bddhMvq3szH1DpMQ6y15l3JNQHRe5fECRRZUJ75eyWWLyyzGvRMwN2zD8eXEfw4C(YchzP9B15nrSCrQX(Kbxx(XlOb(zUY6bigEhclleCvrk8Hi3AN6d3qd2zYj4TwH(tEnIGT(qR2NCYEKyvtFyn9w1eI(3ufSX7zeRU(73)(mpJoWmpKLcY0PC5mpONRMsY0KXh9USRY8m)fkbYXmpsL1fN97)AMxb(nVXJjUfZlrYaqKghEjZ7)ax6NNq(0IWXrDSA1O97OfdmtueS8K3dqHubJhMx5cR4GghicEuL1dSky2ga84qv5v4F7XzVH23DkEFJFWKdt1BHh9NAC)NQg3FQW9)nkCl2b)WkRDdzM))sDBHA6FKL1(SIMED)SEvee(VkvVhaDXlCXGdsyCfLrVgzX74EhKs52lJwo6Z9x9XtVOMs5A63RPiUDhllR27xrm88safTBIBB3)6d`

**Comment by InfusOnWoW** [comment]
This is a bug in LibCustomGlow, which is fixed by https://github.com/Stanzilla/LibCustomGlow/pull/11

**Comment by Loradim** [comment]
Awesome, that you could track down the source.
Thanks a lot for your effort!
